core/services/head_tracker.go

```diff
@@ -42,23 +42,83 @@ var (
 		Help:    "How long it took to execute all callbacks (ms) histogram",
 		Buckets: []float64{50, 100, 250, 500, 1000, 2000, 5000, 10000, 15000, 30000, 100000},
 	})
+	promNumHeadsDropped = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "head_tracker_num_heads_dropped",
+		Help: "The total number of heads dropped",
+	})
```

Comment:
NOTE: This needs an alert added here: https://github.com/smartcontractkit/infra/blob/master/tools/cortex/mgnt/staging/config/ec2/chainlink.rules.yml